.vscode/launch.json

```diff
@@ -5,6 +5,12 @@
 {
 	"version": "0.2.0",
     "configurations": [
+      {
+        "name": ".NET Core Attach",
```

Comment:
@gregg-miskelly is there a way to provide a base input to pick process? Aka, i'd love to filter this to our language server process by default when this is run (rzls.exe)

Reply:
I don't believe so

Reply:
😢

Reply:
I guess I should say: if there is **only one** rzls.exe process, you can attach by process name. by removing the `processId` element, and instead adding `processName`. But if there are multiple processes that will fail.

Reply:
I see, ya unlikely there'd be only one since this would be running in experimental mode (one for the host VSCode)